scipy/stats/tests/test_stats.py

```diff
@@ -7126,7 +7127,7 @@ def test_fived(self, sim_type, obs_stat, obs_pvalue):
         assert_approx_equal(stat, obs_stat, significant=1)
         assert_approx_equal(pvalue, obs_pvalue, significant=1)
 
-    @pytest.mark.slow
+    @pytest.mark.xslow
```

Comment:
Silly question--why do we sometimes use `xslow` and other times use `xslow()`? Is the difference irrelevant?

Reply:
Not silly. `xslow` could be a decorator factory, in which case I'd think it would need the `()`. But both seem to work, so I'm not sure what's going on. I'm guessing that some of us have used one or the other without being careful to see how it is supposed to be used.

Reply:
> Not silly. `xslow` could be a decorator factory, in which case I'd think it would need the `()`. But both seem to work, so I'm not sure what's going on. I'm guessing that some of us have used one or the other without being careful to see how it is supposed to be used. I'll look into it.

The pytest MarkDecorator class actually implements some magic so that they can be used as either decorators or decorator factories. See the pytest documentation [here](https://github.com/pytest-dev/pytest/blob/47df71d23ff13bd083b21ffcf839bd11169b42fc/src/_pytest/mark/structures.py#L284-L302).

Reply:
I figured they must have made some magic if both of them were working! Thanks!